teeracle/src/lib.rs

```diff
@@ -163,6 +215,7 @@ pub mod pallet {
 			let sender_index = <pallet_teerex::Pallet<T>>::enclave_index(sender);
 			let sender_enclave = <pallet_teerex::Pallet<T>>::enclave(sender_index)
 				.ok_or(pallet_teerex::Error::<T>::EmptyEnclaveRegistry)?;
+			// Todo: Never checks data source len
```

Comment:
[Really just optional]
```suggestion
			// TODO: Never checks data source len
```